sfcli/SmartFace.Cli/Commands/SubCamera/DeleteCameraCmd.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Threading.Tasks;
+using McMaster.Extensions.CommandLineUtils;
+using SmartFace.Cli.Core.ApiAbstraction;
+
+namespace SmartFace.Cli.Commands.SubCamera
+{
+    [Command(Name = "delete", Description = "Delete a camera")]
+    public class DeleteCameraCmd
+    {
+        [Option("-i|--id", "Id of camera to delete.", CommandOptionType.SingleValue)]
+        public (bool HasValue, Guid Value) CameraId { get; }
+
+        private ICamerasRepository Repository { get; }
+
+        public DeleteCameraCmd(ICamerasRepository repository)
+        {
+            Repository = repository;
+        }
+
+        protected virtual async Task OnExecuteAsync(IConsole console)
+        {
+            if (!CameraId.HasValue)
```

Comment:
Is not possible to use required attribute like in AddCameraCmd?

Reply:
yes, added

Reply:
then I think this condition can be removed and this should be handled internally by framework

Reply:
yeah, I was testing if this is really the case

Reply:
and maybe it is also sufficient to declare CameraId only as pure Guid

Reply:
yes, done, it works as expected